.gitlab/ci/sdk.yml

```diff
@@ -6,10 +6,10 @@
     - bash docker-sdk.sh
     - docker push "$DOCKER_IMAGE"
 
-build-sdk:
+deploy-sdk-x86-64:
+  stage: build
```

Comment:
Having twice the same name caused a merge of the job variables, that's why this job ended up in the `deploy` stage. Adding it explicitly to the `build` stage still used the `deploys` script causing trouble.